### DISCUSSION TITLE
Use of CMake config under windows, with OpenSSL

### DISCUSSION THREAD

**Discussion by ntwerdochlib** [Yubico/libfido2#664] [Q&A] [answered]
The CMake config for libfido2 seems to support inclusion into windows projects well?  Especially if you want to make use of an existing OpenSSL library that is already used by the project.

The windows build powershell script implies that the project is really tied to LibreSSL?

Wondering if I am just not seeing the way to make it work or if there is just not much use under windows?

**Comment by LDVG** [comment, marked as answer]
> The CMake config for libfido2 seems to support inclusion into windows projects well? Especially if you want to make use of an existing OpenSSL library that is already used by the project.

In theory, it should be possible to link against OpenSSL on Windows but there might be some work required to accommodate that in the build system.

> The windows build powershell script implies that the project is really tied to LibreSSL?

The script is mostly there for CI and release purposes and should not be considered the only way to build libfido2 on Windows.

> Wondering if I am just not seeing the way to make it work or if there is just not much use under windows?

We'd be happy to try to make things work better for projects wishing to make use of an existing OpenSSL build and any help is more than welcome.
